tests/unit/test_forms.py

```diff
@@ -69,12 +69,12 @@ def test_good_passwords(self, password):
         [
             (
                 "qwerty",
-                ("This is a top-10 common password. Add another word or two. "
+                ("This is a top-10 common password. Add another word or two."
                  "Uncommon words are better."),
             ),
             (
                 "bombo!b",
-                ("Password is too easily guessed. Add another word or two. "
+                ("Password is too easily guessed. Add another word or two."
```

Comment:
This doesn't seem right, there should be a space between the two sentences?

Reply:
You are right. I made a mistake here. Sorry about that.